src/pages/app/Documents.svelte

```diff
@@ -181,7 +181,7 @@
             >
           {/if}
           {#if $orgsAndUsers.loggedIn && !$orgsAndUsers.isVerified}
-            <a href="https://airtable.com/app93Yt5cwdVWTnqn/pagogIhgB1jZTzq00/form" target="_new">
+            <a href="https://airtable.com/app93Yt5cwdVWTnqn/pagogIhgB1jZTzq00/form?prefill_MR+User+Name={$orgsAndUsers.me.username}&prefill_MR+User+Account+URL=https://accounts.muckrock.com/users/{$orgsAndUsers.me.username}/&prefill_MR+Organization+Name={$orgsAndUsers.me.organization.slug}&prefill_MR+Organization+Account+URL=https://accounts.muckrock.com/organizations/{$orgsAndUsers.me.organization.slug}" target="_new">
```

Comment:
Is there a way to build this string in a more manageable way in case we need to change parameters in the future?  Maybe make an object with the keys and values and then convert it to a query string?  Not a show stopped, but might be nice, especially if we want to do similar things in other places.

Reply:
We could pull this whole URL up to the top of the component and build it as a variable there. @duckduckgrayduck I can walk you through this if you need. 

But also, this entire component and section of the codebase is only going to live for another two or three months max.

Reply:
Is the idea here to write a function at the top that takes the various prefill arguments and generates the URL? Something like

function generateAirtableLink() {
    const username = $orgsAndUsers.me.username;
    const orgSlug = $orgsAndUsers.me.organization.slug;
    return `https://airtable.com/app93Yt5cwdVWTnqn/pagogIhgB1jZTzq00/form?prefill_MR+User+Name=${username}&prefill_MR+User+Account+URL=https://accounts.muckrock.com/users/${username}/&prefill_MR+Organization+Name=${orgSlug}&prefill_MR+Organization+Account+URL=https://accounts.muckrock.com/organizations/${orgSlug}`;
  }

Reply:
That would work. Or you could use a computed property, like this: https://svelte.dev/docs/svelte-components#script-3-$-marks-a-statement-as-reactive

Do whatever makes you most comfortable at this point. I'd rather get this in than split hairs over implementation.